diff --git a/node_modules/webpack-dev-server/client/overlay.js b/node_modules/webpack-dev-server/client/overlay.js
index 1524861..4ca4027 100644
--- a/node_modules/webpack-dev-server/client/overlay.js
+++ b/node_modules/webpack-dev-server/client/overlay.js
@@ -183,6 +183,13 @@ var createOverlay = function createOverlay(options) {
    * @param {'build' | 'runtime'} messageSource
    */
   function show(type, messages, trustedTypesPolicyName, messageSource) {
+    if (messages.every(reason => {
+      const msg = typeof reason === 'string' ? reason : reason.message;
+      return msg && msg.includes('ResizeObserver');
+    })) {
+      hide();
+      return;
+    }
     ensureOverlayExists(function () {
       headerElement.innerText = messageSource === "runtime" ? "Uncaught runtime errors:" : "Compiled with problems:";
       messages.forEach(function (message) {
